guides/doc-Configuring_Load_Balancer/topics/Installing_the_Load_Balancer.adoc

```diff
@@ -40,7 +40,7 @@ You must configure sticky session on TCP port 443 to request yum metadata for RP
 //| Anaconda | 8000 | TCP | roundrobin | port 8000 on all {SmartProxies}
 | HTTPS | 443 | TCP | source | port 443 on all {SmartProxyServer}s
 | RHSM | 8443 | TCP | roundrobin | port 8443 on all {SmartProxyServer}s
-| AMQP | 5647 | TCP | roundrobin | port 5647 on all {SmartProxyServer}s
+| AMQP | 5{ProductVersion} | TCP | roundrobin | port 5{ProductVersion} on all {SmartProxyServer}s
```

Comment:
This has made my morning. 
The most Melanie of sed's performed by spetrosi :)

Reply:
Ahahaha oh my, will fix now